modules/analyze_basic_content_info/src/Plugin/Analyze/ContentInfo.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\analyze_basic_content_info\Plugin\Analyze;
+
+use Drupal\analyze\AnalyzePluginBase;
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\FieldableEntityInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Analyze plugin to display Basic data.
+ *
+ * @Analyze(
+ *   id = "content_info",
+ *   label = @Translation("Basic Content Entity Reports"),
+ *   description = @Translation("Provides basic statistics about content for Analyzer.")
+ * )
+ */
+final class ContentInfo extends AnalyzePluginBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Creates the plugin.
+   *
+   * @param array $configuration
+   *   Configuration.
+   * @param $plugin_id
+   *   Plugin ID.
+   * @param $plugin_definition
+   *   Plugin Definition.
+   * @param \Drupal\Core\Entity\EntityFieldManagerInterface $entityFieldManager
+   *   Statistics service.
+   */
+  public function __construct(
+    array $configuration,
+    $plugin_id,
+    $plugin_definition,
+    protected EntityFieldManagerInterface $entityFieldManager,
+  ) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_field.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function renderSummary(EntityInterface $entity): array {
+    return [
+      '#type' => 'table',
+      '#header' => [['data' => 'Basic Info', 'colspan' => 2, 'class' => ['header']]],
+      '#rows' => [
+        ['data' => ['Word count', $this->getWordCount($entity)]],
+        ['data' => ['Image count', $this->getImageCount($entity)]],
+      ],
+      '#attributes' => [
+        'class' => ['basic-data-table'],
+        'style' => ['table-layout: fixed;'],
+      ],
+    ];
+  }
+
+  /**
+   * Helper to calculate a rough word count for the entity.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to calculate a word count for.
+   *
+   * @return int
+   *   The counted words. Defaults to 0.
+   */
+  private function getWordCount(EntityInterface $entity): int {
+    $return = 0;
+    $countables = [];
+
+    if ($entity instanceof FieldableEntityInterface) {
+      foreach ($this->getFieldDefinitionsByType($entity, [
+        'string',
+        'string_long',
+        'text_long',
+        'text_with_summary',
+        'text',
+      ]) as $definition) {
+        if ($name = $definition->getName()) {
+          if ($name !== 'timezone') {
+            $value = $entity->get($name)->getValue();
```

Comment:
You could make use of the field type's main property so you are less reliant on `value`?

Reply:
The other option is to use the rendered version of an entity, which would automatically track through things like paragraphs etc?

Reply:
I don't understand these comments I'm afraid. Is there a problem with using getValue() on a field to get its value?

Reply:
Currently it would always work with value, since you only get text fields in the conditional before. 

But this will not work with Paragraphs for instance that DXPR uses heavily, since that would be an entity revision reference field with a paragraph. They would be loaded via target_id and revision_id etc. There are other examples of referenced fields where you would need to do word count I guess.

Also this logic wouldn't take notice of visibility of fields. If you for instance have text_long or string_long field that are just being used as meta data to for instance improve searching, they should probably not be counted?

Getting the rendered version and removing tags and then counting the words are probably the best version of solving this - of course depending on what the customer wants, but that would at least count the words they are seeing.

Reply:
I have changed this to rendered entity and it works kind off ok - it counts "Add new comment" for instance if you have comments on and it counts the title (which might not be bad).

I also fixed so it counts based on translation now as well.